ipatests/azure/scripts/setup_containers.py

```diff
@@ -2,7 +2,7 @@
 import os
 import subprocess
 
-import docker
+import docker  # pylint: disable=import-error
```

Comment:
`python3-docker` is the 'lint' dependency from the spec.

Reply:
Ah, I see. I've removed the change.